Shell/SymCounter.cpp

```diff
@@ -166,7 +166,7 @@ void SymCounter::count (Formula* f,int polarity,int add)
 
     case BOOL_TERM: {
       TermList ts = f->getBooleanTerm();
-      if (!ts.isTerm()) return;
+      if (!ts.isTerm()) return;     
```

Comment:
various bits of whitespace - delete to minimise diff

Reply:
Done